### PR TITLE
security-vulnerabilities: add more info about base images

### PR DESCRIPTION
**Pull request by howardjohn**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/releases/security-vulnerabilities/index.md [hunk 1]
```diff
@@ -87,3 +88,18 @@ branches.
 As much as possible this announcement will be actionable, and include any mitigating steps customers can take prior to
 upgrading to a fixed version. The recommended target time for these announcements is 16:00 UTC from Monday to Thursday.
 This means the announcement will be seen morning Pacific, early evening Europe, and late evening Asia.
+
+## Base Images
+
+Istio offers two sets of docker images, based on `ubuntu` (default) and based on `distroless` (see [Harden Docker Container Images](docs/ops/configuration/security/harden-docker-images/)).
+These base images occasionally have CVEs.
+The Istio security team has automated scanning to ensure base images are kept free of CVEs.
```
Comment (ericvn) [inline content/en/docs/releases/security-vulnerabilities/index.md:96]: product security?

File: content/en/docs/releases/security-vulnerabilities/index.md [hunk 2]
```diff
@@ -87,3 +88,18 @@ branches.
 As much as possible this announcement will be actionable, and include any mitigating steps customers can take prior to
 upgrading to a fixed version. The recommended target time for these announcements is 16:00 UTC from Monday to Thursday.
 This means the announcement will be seen morning Pacific, early evening Europe, and late evening Asia.
+
+## Base Images
+
+Istio offers two sets of docker images, based on `ubuntu` (default) and based on `distroless` (see [Harden Docker Container Images](docs/ops/configuration/security/harden-docker-images/)).
```
Comment (ericvn) [inline content/en/docs/releases/security-vulnerabilities/index.md:94]:
```suggestion
Istio offers two sets of docker images, one based on `Ubuntu` (default) and another based on `distroless` (see [Harden Docker Container Images](docs/ops/configuration/security/harden-docker-images/)).
```

File: content/en/docs/releases/security-vulnerabilities/index.md [hunk 3]
```diff
@@ -87,3 +88,18 @@ branches.
...
+These base images occasionally have CVEs.
+The Istio security team has automated scanning to ensure base images are kept free of CVEs.
+
+When CVEs are detected in our images, new images are automatically built and used for all future builds.
+Additionally, the security team analyzes the vulnerabilities to see if they are exploitable in Istio directly.
```
Comment (ericvn) [inline content/en/docs/releases/security-vulnerabilities/index.md:99]: product security

File: content/en/docs/releases/security-vulnerabilities/index.md (hunk 1 above)
Comment (howardjohn) [reply content/en/docs/releases/security-vulnerabilities/index.md:96]: rest of page uses "Istio security team". Want me to change them all?

Comment (ericvn) [reply content/en/docs/releases/security-vulnerabilities/index.md:96]: Nope. Sounds like that's been the norm, and I had missed that.

**Review by ericvn — APPROVED** [review]

**Pull request merged**